### ISSUE TITLE
Incorrect coefficient numbers in Table 3

### ISSUE THREAD

**Issue by andrewfoote** [larsvilhuber/MobZ#44]
Table 3 has the following coefficients for columns 3 and 4: -0.8748
But the correct value (which is also in Table 3 log) is -0.8818.

The standard errors are also incorrect.

**Comment by larsvilhuber** [comment]
Yeah, that's why for AEA we would not accept such a log file/program. 

Can you annotate the program with where each cell (row/column) of Table 3 is pulled from?
